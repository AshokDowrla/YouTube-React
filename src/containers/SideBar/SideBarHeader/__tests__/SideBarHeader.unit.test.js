import React from 'react';
import {shallow} from 'enzyme';
import SideBarHeader from '../SideBarHeader';

describe('SideBarHeader', () => {
  test('renders without title', () => {
    const wrapper = shallow(<SideBarHeader/>);
    expect(wrapper).toMatchSnapshot();
  });


  test('renders with an empty title', () => {
    const wrapper = shallow(<SideBarHeader title="" />);
    expect(wrapper).toMatchSnapshot();
  });

  test('renders with a title', () => {
    const wrapper = shallow(<SideBarHeader title="ust a title" />);
    expect(wrapper).toMatchSnapshot();
  });


});