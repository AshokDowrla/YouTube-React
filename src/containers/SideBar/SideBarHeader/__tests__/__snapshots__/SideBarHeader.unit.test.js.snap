// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SideBarHeader renders with a title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  >
    UST A TITLE
  </MenuHeader>
</MenuItem>
`;

exports[`SideBarHeader renders with an empty title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;

exports[`SideBarHeader renders without title 1`] = `
<MenuItem>
  <MenuHeader
    className="side-bar-header"
  />
</MenuItem>
`;
